src/OpenTelemetry.AutoInstrumentation.Native/cor_profiler.cpp

```diff
@@ -156,35 +175,27 @@ HRESULT STDMETHODCALLTYPE CorProfiler::Initialize(IUnknown* cor_profiler_info_un
         }
     }
 
-    // Initialize ReJIT handler and define the Rewriter Callback
-    auto callback = [this](RejitHandlerModule* mod, RejitHandlerModuleMethod* method) {
-        return this->CallTarget_RewriterCallback(mod, method);
-    };
-
-    rejit_handler = new RejitHandler(this->info_, callback);
-
-    const bool instrumentation_enabled_by_default = AreInstrumentationsEnabledByDefault();
+    // get ICorProfilerInfo10 for >= .NET Core 3.0
+    ICorProfilerInfo10* info10 = nullptr;
```

Comment:
Separate GH issue:
Remove references to ICorProfilerInfo10  in whole repository as we do not support .NET Core 3.0.
It could be replaced to ICorProfilerInfo12 to reduce number of checked interfaces.
Also, it will be great to move fetching code to the place where ICorProfilerInfo7 is fetched.